docs/add_custom_features.md

```diff
@@ -0,0 +1,66 @@
+# Add custom features for custom analysis
+
+If you want to perform custom analysis with your custom features that are not supported in the original task processors, and these features are only related to one particular system/dataset, you can define the custom features and analysis in the `metadata` section in the output JSON file. 
+
+Here is the template of the output JSON file with custom metadata.
```

Comment:
This is nice, but it might be even more clear if you separately add one example of a discrete feature and one example of a continuous feature (and explicitly explain that you're doing so in the text).